### ISSUE TITLE
Leaderboard Intent ignoring requested collection

### ISSUE THREAD

**Issue by recioalex** [playgameservices/android-basic-samples#215]
When requesting the leaderboard intent with:

Games.Leaderboards.getLeaderboardIntent(mgapiclient,leaderBoard,LeaderboardVariant.TIME_SPAN_WEEKLY,LeaderboardVariant.COLLECTION_PUBLIC)

The time span works... but It allways shows the SOCIAL leaderboard and not the PUBLIC...

**Comment by muafakul** [comment]
yes that was bug ,, baddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddd

**Comment by indraazimi** [comment]
+1

**Comment by recioalex** [comment]
In google games api v9.8 the isue its still there....

**Comment by uriel-frankel** [comment]
+1

**Comment by claywilkinson** [comment]
I've passed this along to the development team to fix.
